docs/flashbots-protect/rpc/quick-start.md

```diff
@@ -1,25 +1,29 @@
 ---
 title: quick start
 ---
-**Please note that Flashbots Protect RPC is currently in public beta. Your transactions may occasionally time out and not be mined!**
+**Please note that Flashbots Protect RPC is currently in public beta. Your transactions may occasionally time out and not be included!**
 
 Flashbots Protect RPC allows regular users to easily submit their transactions to the Flashbots Auction by using a custom RPC endpoint in their wallet. Everything should be the same for users, except transactions are sent to Flashbots, and then directly to miners, instead of the public mempool.
 
 There are a few key benefits to using the Flashbots RPC endpoint:
+
 - **Frontrunning protection:** your transaction will not be seen by hungry sandwich bots in the public mempool.
-- **No failed transactions:** your transaction will only be mined if it doesn't include any reverts, so you don't pay for failed transactions. Note: your transaction could be uncled, emitted to the mempool, and then included on-chain.
+- **No failed transactions:** your transaction will only be included if it doesn't include any reverts, so you don't pay for failed transactions. Note: your transaction could be uncled, emitted to the mempool, and then included on-chain.
 - **Priority in blocks:** transactions sent via Flashbots are mined at the top of blocks, giving them priority.
 
 ## Key considerations
-Before you get started here are a few things to be mindful of
+
+Before you get started here are a few things to be mindful of:
+
 - **You can find the status of your transaction on Etherscan.** Etherscan has a nice interface for viewing the status of your transaction from our [status API](/flashbots-protect/rpc/status-api).
-- We will try to resubmit your transaction for 25 blocks after which point it is considered “expired” and will be dropped.
-- Transactions under 42,000 gas, such as simple ether transfers, are rejected by the Flashbots relay. As a result, we will forward these to the public mempool instead.
+- We will try to include your transaction for 3 days, after which point it is considered “expired” and will be dropped.
+- Transactions under 42,000 gas, such as simple ether transfers, do not need front-running or revert protection. We will forward these to the public mempool instead.
```

Comment:
We will forward these to the public mempool instead to provide the fastest execution possible.